shared/profile/index.desktop.js

```diff
@@ -505,6 +507,14 @@ const styleFolderIcon = {
   textAlign: 'center',
 }
 
+const styleMeta = {
```

Comment:
🐑 🐑  if this is repeated often enough it might make sense to factor it into an `<OpenTeamTag />`

Reply:
Thanks!  Thought about that but the top/left margins are usually different each time.